zerver/webhooks/jotform/doc.md

```diff
@@ -0,0 +1,16 @@
+Get Zulip notifications for your JotForm responses!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+   By default, the integration will use the name of ***JotForm-Response*** as topic. Make sure to ***not use colon*** as part of form's questions.
+
+1. Go to your Jotform **Form Builder**, and click on **Settings** tab.
+
+1. Click on **INTEGRATIONS**, search for **WebHooks** and select it.
+
+1. Add your **bot-url** to the field and click the **COMPLETE INTEGRATION** button.
```

Comment:
Why is this spelled `bot-url`?

Reply:
Sorry, I will change this to ***stream-url***.